### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Mime Maps_Keyboard: When we press enter on edit button, keyboard focus is going to the first control of page instead of going edit field.

### ISSUE THREAD

**Issue by Accessibility56** [Microsoft/IIS.WebManager#142]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #.NETWebsites; #Keyboard; #WCAG2.4.3; #C+E-H; #Chrome; #Edge

**Environment Details:**
**URL**: https://manage.iis.net/web-sites
**Browser Details:** Edge(42.17134.1.0), Chrome(68.0.3440.106)
Keyboard

**Repro Steps:**
1.Download and install "Microsoft IIS Administration" application.
2.Enable Internet information Services.
3.Open "Microsoft IIS" application using the URL: https://manage.iis.net/.
4.Tab till Web Server and press enter.
5.Tab till Mime Maps and press enter.
6.Tab till "Edit" in grid and press enter.
7.Verify focus order after pressing enter on edit field.

**Actual Result:**
 When we press enter on edit button, keyboard focus is going to the first control of page instead of going edit field.

**Expected Result:**
Keyboard focus should be inside the control and should not start from first control of page when we activate any control.

**User Impact:** 
Users get confuse if focus order is not as expected.

**Attachment:**
[Bug_142_Central Certificate Store_Focus Order issue.pptx](https://github.com/Microsoft/IIS.WebManager/files/2330822/Bug_142_Central.Certificate.Store_Focus.Order.issue.pptx)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Yashi1201** [comment]
**Github tags:** #A11ySev2;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
